src/color/hexToRgb.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Converts a HTML5 color value (string) to RGB values.
+ * Conversion formula:
+ * - split the string; "#RRGGBB" into RGB components
+ * - convert the HEX value into RGB values
+ * @see the color input type of HTML5 forms
+ */
+const hexToRgb = (value) => {
```

Comment:
I think this needs to work for rgbA hex values, (even more so if we remove the alpha parameter from the color() op), 
we already have such a thing here https://github.com/jscad/OpenJSCAD.org/blob/V2/packages/utils/img/src/colorConversions.js
This is one of the things I mentionned needing cleanup after we merge Openjscad.org & csg, so we can deal with it then ?

Reply:
Changed to accept an string notiation with RGB or RGBA hex values.